### PR TITLE
Various minor fixes for clarity

### PR DESCRIPTION
**Pull request by mikemaccana**

### CODE REVIEW

**Comment by xizhao** [comment]
Thanks for the PR.  I'm swamped and trying to get to thing by this weekend -- in the meantime redrafting the FAQ and will take these comments in.

File: index.html [hunk 1]
```diff
@@ -42,20 +42,20 @@ <h1>The Commons Clause<span class="dot">.</span></h1>
 
       <h2>FAQ</h2>
       <h3>What is Commons Clause?</h3>
-      <p>The Commons Clause is a license condition drafted by <a>Heather Meeker</a> that applies a minimal-form commercial restriction on top of an existing open source license to transition the project to a source-available licensing scheme.  It was designed to a minimal and readable “textual” change to meet business and legal requirements.</p>
-      <p>This Clause is not intended to be applied against at-scale existing open source projects, but incrementally on top of commercial counterparts that need to be transitioned to source-availability to satisfy urgent business or legal requirements.</p>
+      <p>The Commons Clause is a license change drafted by <a>Heather Meeker</a> that replaces an open source license, allowing all  permissions of the original license to remain except the ability to sell the software. It was designed to a minimal and readable “textual” change to meet specific business and legal requirements.</p>
```
Comment (xizhao) [inline index.html]: Nice, I like this clarification.

**Comment by xizhao** [comment]
Okay, I like a lot of this.  Incorporated in #6.

**Comment by mikemaccana** [comment]
Thanks Kevin. I'd also like to apologise for being antagonistic earlier - I assumed bad intentions where there wasn't any. Sorry. 🙂

**Pull request closed without merging**